src/integrationTest/java/uk/gov/hmcts/darts/transcriptions/controller/TranscriptionControllerAdminGetTranscriptionIntTest.java

```diff
@@ -333,6 +338,173 @@ void getTransactionsForUserBeyondOrEqualToDateNoUserSpecified() throws Exception
+
+        SearchTranscriptionDocumentRequest request = new SearchTranscriptionDocumentRequest();
+        request.setCaseNumber(transcriptionDocumentResults.getFirst().getTranscription().getHearing().getCourtCase().getCaseNumber());
+
+        MvcResult mvcResult = mockMvc.perform(post(ENDPOINT_DOCUMENT_SEARCH)
+                                                  .header("Content-Type", "application/json")
+                                                  .content(objectMapper.writeValueAsString(request)))
+            .andExpect(status().is2xxSuccessful())
+            .andReturn();
+
+        assertEquals(200, mvcResult.getResponse().getStatus());
+
+        JSONAssert.assertEquals(
+            """
+                [
+                   {
+                     "transcription_document_id": 3,
+                     "transcription_id": 3,
+                     "courthouse": {
+                       "display_name": "COURT NAME:1POST COURTHOUSE"
+                     },
+                     "hearing": {
+                       "hearing_date": "%DATE_3%"
+                     },
+                     "is_manual_transcription": false,
+                     "is_hidden": false,
+                     "case": {
+                       "id": 1,
+                       "case_number": "%CASE_NUMBER%"
+                     }
+                   },
+                   {
+                     "transcription_document_id": 2,
+                     "transcription_id": 2,
+                     "courthouse": {
+                       "display_name": "COURT NAME:1POST COURTHOUSE"
+                     },
+                     "hearing": {
+                       "hearing_date": "%DATE_2%"
+                     },
+                     "is_manual_transcription": false,
+                     "is_hidden": false,
+                     "case": {
+                       "id": 1,
+                       "case_number": "%CASE_NUMBER%"
+                     }
+                   }
+                 ]
+                """
+                .replace("%CASE_NUMBER%", request.getCaseNumber())
+                .replace("%DATE_2%", OffsetDateTime.now().plusDays(4).format(DateTimeFormatter.ISO_LOCAL_DATE))
+                .replace("%DATE_3%", OffsetDateTime.now().plusDays(8).format(DateTimeFormatter.ISO_LOCAL_DATE)),
+            mvcResult.getResponse().getContentAsString(), JSONCompareMode.STRICT);
+    }
+
+    @Test
+    void documentSearch_notSuperUser_shouldSeeHiddenDocuments() throws Exception {
+        List<TranscriptionDocumentEntity> transcriptionDocumentResults = transcriptionDocumentStub.generateTranscriptionEntities(4, 1, false, true, false,
+                                                                                                                                 true);
+
+        superAdminUserStub.givenUserIsAuthorised(userIdentity);
+        when(userIdentity.userHasGlobalAccess(Set.of(SecurityRoleEnum.SUPER_USER))).thenReturn(false);
```

Comment:
The test title says 'notSuperUser' but the role used here is Super User, is the test title wrong?